camel/types/enums.py

```diff
@@ -89,6 +89,12 @@ class ModelType(UnifiedModelType, Enum):
     REKA_FLASH = "reka-flash"
     REKA_EDGE = "reka-edge"
 
+    # Cohere models
+    COHERE_COMMAND_R = "command-r"
+    COHERE_COMMAND_LIGHT = "command-light"
+    COHERE_COMMAND = "command"
+    COHERE_COMMAND_NIGHTLY = "command-nightly"
```

Comment:
from https://docs.cohere.com/v2/docs/models more models like `command-r-plus` could be supported